### ISSUE TITLE
Conway's Game Of Life

### ISSUE THREAD

**Issue by prosauer** [joergbrech/Modellbildung-und-Simulation#27]

**Comment by MartinKliemank** [comment]
+ coole Bilder wie Slider-Kanone ect. am Ende

**Comment by MartinKliemank** [comment]
@joergbrech  Wie genau war die Aufgabe nochmal gedacht? Nur Regeln vorstellen und vll. Hinweise geben, wie man vorgeht, oder ganze Musterlösung?

Ist es ok, Grafiken aus Wikipedia zu nutzen, oder sollte ich die lieber wieder selber bauen? (Ist ja auch nicht soo viel Aufwand)

**Comment by joergbrech** [comment]
@MartinKliemank Eine Musterlösung soll es hier eigentlich zu keinen Aufgaben geben. Ich dachte an einen ähnlich hierarchischen Aufbau wie die Verkehrssimulationsaufgabe oder die Bodensimulationsaufgabe. In etwa 

 - Aufgabe 1: Regeln umsetzen, u.U. mit unit test
 - Aufgabe 2-x: Unterschiedliche Erweiterungen wie zB. unterschiedliche Arten der Randbedingungen, Regelabweichungen etc. Hier könnte man auch verschiedene Anfangsbedingungen generieren lassen, wie bestimmte statische Objekte oder oszillierende Objekte.
 - Letzte Aufgabe(n): Postprocessing/Analyse der Simulation. Das könnten zum Beispiel Populationsanalysen zu unterschiedlichen Randbedingungen sein.

In diesem Rahmen hast du künstlerische Freiheit und kannst dich gerne austoben ;)

Grafiken aus Wikipedia kannst du gerne dann benutzen, wenn die Lizenz es zulässt. Alle Bilder auf Wikipedia sind auch mit Lizenzangaben auf Wikimedia Commons hinterlegt. Achte darauf, ob eine *Author Attribution* verlangt wird oder nicht.

**Comment by MartinKliemank** [comment]
@joergbrech Ok, gut, das klingt nach einem sinnvollen Aufbau. 
Was meinst du genau mit Randbedingungen? Das Verhalten am Rand vom Spielfeld, oder noch was anderes?

**Comment by joergbrech** [comment]
Ja, genau das Verhalten am Rand vom Spielfeld. Das war aber auch nur als Beispiel gedacht, falls sich daraus eine interessante Aufgabe formulieren lässt.

**Comment by joergbrech** [comment]
Martins bisherige Arbeiten hierzu: https://github.com/joergbrech/Modellbildung-und-Simulation/tree/gameOfLife
